gradle/lib/dependabot/gradle/update_checker/version_finder.rb

```diff
@@ -184,10 +185,8 @@ def dependency_metadata(repository_details)
             begin
               response = Excon.get(
                 dependency_metadata_url(repository_details.fetch("url")),
-                user: repository_details.fetch("username"),
-                password: repository_details.fetch("password"),
                 idempotent: true,
-                **SharedHelpers.excon_defaults
+                **Dependabot::SharedHelpers.excon_defaults(headers: repository_details.fetch("auth_details"))
```

Comment:
The Dependabot::SharedHelpers sets up default header 'User-agent', and merges the headers given as argument in the excon_defaults method. In case of an empty array (if no auth details were found) none will be added, but there will still be headers on the request made.

Reply:
Perhaps `auth_headers` would be a better term.

Reply:
I have made an update renaming auth_details auth_headers, I hope that better reflects what it is.